code/modules/power/supermatter/supermatter.dm

```diff
@@ -533,6 +533,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 		if(prob(15) && power > POWER_PENALTY_THRESHOLD)
 			supermatter_pull(src, power/750)
+
+		if(power > POWER_PENALTY_THRESHOLD && prob(3) || power > SEVERE_POWER_PENALTY_THRESHOLD && prob(5) || power > CRITICAL_POWER_PENALTY_THRESHOLD && prob(10))
```

Comment:
What if we used BZ concentration or radiation level to spawn slimes instead of POWER. co2 setups are fun as they are while the BZ SM is pretty boring. Also it's caped by BZ supply so you will have to get some from Cargonia or make a farm before you'll be able to get to the top slimes. Also BZ SMs are pretty rare so it won't happen every round or with a usual delamination.